src/openssl.h

```diff
@@ -161,7 +161,7 @@
 #endif
 
 #if (OPENSSL_VERSION_NUMBER >= 0x01010100fL && !defined(OPENSSL_NO_AES)) || \
-    (defined(LIBSSH2_WOLFSSL) && defined(HAVE_AESGCM))
+    (defined(LIBSSH2_WOLFSSL) && defined(WOLFSSL_AESGCM_STREAM))
```

Comment:
While our AES-GCM stream support should depend on having AES-GCM in general enabled, I'd recommend checking for both here to be safe, ie:
`(defined(HAVE_AESGCM) && defined(WOLFSSL_AESGCM_STREAM))`

Reply:
Restored the previous check and now checking for both.